sheetScript.gs

```diff
@@ -11,6 +11,17 @@ var debugFlag_dontUpdateTimestamp = false;
 var debugFlag_dontUpdatePlaylists = false;
 var debugFlag_logWhenNoNewVideosFound = false;
 
+
+// Define reserved Rows and Columns (zero-based)
+var reservedTableRows = 2;        // Start of the range of the PlaylistID+ChannelID data
+var reservedTableColumns = 4;     // Start of the range of the ChannelID data
```

Comment:
These were zero-based before, so should stay as
```
var reservedTableRows = 3;        // Start of the range of the PlaylistID+ChannelID data
var reservedTableColumns = 5;     // Start of the range of the ChannelID data
```
We want reservedTableRows to be row 4 of the sheet, and reservedTable Columns to be column F of the sheet, the start of user data